models/dex/dex_sandwiches.sql

```diff
@@ -0,0 +1,348 @@
+{{ config(
+        alias ='sandwiches',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain', 'sandwiched_pool', 'frontrun_tx_hash', 'frontrun_taker', 'frontrun_index', 'currency_address'],
+        post_hook='{{ expose_spells(\'["ethereum", "bnb", "avalanche_c", "gnosis", "optimism", "arbitrum", "fantom", "polygon"]\',
+                                "sector",
+                                "dex",
+                                \'["hildobby"]\') }}'
+        )
+}}
+
+WITH trades AS (
```

Comment:
similar to the other recently reviewed PR, can we leverage the for loop approach here, with dictionary variables?

Reply:
looking closer, now i see the runtime in the ci test, i wonder if we should split this out per blockchain, then rollup as usual. this is a lottt of data for one spell to crunch all these joins and such

Reply:
👍 changed it to the usual structure of chain model that join into a crosschain one